src/mono/netcore/sample/wasm/browser/runtime.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Add copyright header:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

```